static/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -336,9 +336,7 @@ export default class AbstractExternalIssueForm<
   ) => {
     const initialData: {[key: string]: any} = (formFields || []).reduce(
       (accumulator, field: FormField) => {
-        accumulator[field.name] =
-          // Passing an empty array breaks MultiSelect.
```

Comment:
does passing an empty array no longer break multiselect?

Reply:
Yes it works as intended 👍